test/details.test.js

```diff
@@ -129,6 +139,25 @@ describe('vaadin-details', () => {
       details.opened = true;
       expect(content.getAttribute('aria-hidden')).to.equal('false');
     });
+
+    it('should set unique id on the content', () => {
+      const id = 1 + getUniqueId();
+      const container = fixtureSync(`
+        <div>
+          <vaadin-details>
+            <div slot="summary">Summary</div>
+            <input>
+          </vaadin-details>
+          <vaadin-details>
+            <div slot="summary">Summary</div>
+            <input>
+          </vaadin-details>
+        </div>
+      `);
+      const details = container.querySelectorAll('vaadin-details');
+      expect(details[0]._collapsible.id).to.equal(`vaadin-details-content-${id}`);
+      expect(details[1]._collapsible.id).to.equal(`vaadin-details-content-${1 + id}`);
```

Comment:
No need to check for specific number here. 
It is enough to compare the `id` of two elements and check they are different.